### DISCUSSION TITLE
Steps to adopt an Open Source License Compliance Tool

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#114] [OSPO Discussions]
Discussion started via [TODO Slack by Lucius Curado](https://thetodogroup.slack.com/archives/C0GBYUWCB/p1653151012802029).

*Hi all, trying to follow a journey to adopt an Open Source License Compliance Tool (eg. Fossology, ORT) as part of an AWS CI/CD pipeline to support our product teams to follow the company's Open Source policies. Is anyone already doing this who could share experience/documentation?*

**Comment by jeffpaul** [comment]
I looked into tools that integrated with GitHub Actions (as our preferred workflow) and unfortunately I didn't find anything that had an OSS free plan.  If someone does find something that supports an OSS free license, I'd love to hear!

**Comment by juliancoccia** [comment]
This is exactly what SCANOSS resolves. You can see our entire platform here: https://github.com/scanoss

You can also try our CLI:
pip3 install scanoss
scanoss-py scan TARGET

We also have a webhook here:
https://github.com/scanoss/webhook
